geoalchemy2/shape.py

```diff
@@ -43,9 +48,18 @@ def from_shape(shape, srid=-1):
         An integer representing the spatial reference system. E.g. 4326.
         Default value is -1, which means no/unknown reference system.
 
+    ``use_ewkb``
+
+        A boolean to indicate whether to use EWKB or WKB.
+        In EWKB the SRID is included, as a result srid parameter is ignored.
+        Default value is False, which means that WKB is used.
+
     Example::
 
         from shapely.geometry import Point
         wkb_element = from_shape(Point(5, 45), srid=4326)
     """
+    if use_ewkb:
+        return EWKBElement(buffer(shape.wkb))
```

Comment:
I now understand that the above code (with `wkbwriter_defaults` and stuff) won't work because the srid is not passed to the WKBWriter.